org.eclipse.corrosion/OSGI-INF/l10n/bundle.properties

```diff
@@ -22,4 +22,13 @@ command.name.0 = Remove Cargo Builder
 command.name.1 = Implementations
 perspective.name = Rust
 action.label = Toggle Breakpoint
-debugPreferencePage = Debug
\ No newline at end of file
+debugPreferencePage = Debug
+
+#Default templates for rust files
```

Comment:
Are those meant to be internationalized? If not, please move them to another file.

Reply:
Also, isn't it something that would fit better in RustLS directly?

Reply:
Ah ok, I realized you used it as a tweak to load them in the plugin.xml